manifests/windows/install.pp

```diff
@@ -56,7 +57,7 @@
     default => "${::system32}\\cmd.exe"
   }
 
-  if (member($::puppet_agent::service_names, 'puppet')) {
+  if (member($::puppet_agent::params::service_names, 'puppet')) {
```

Comment:
Is this adequate? Given the brittleness of testing in this module, I'm hesitant to make any too-large structural changes, so I'm hoping to do the minimum to fix the bug right now.

Reply:
You want `$::puppet_agent::service_names` there as-is b/c it's what the user passes in, but you'd want to update [install_puppet.bat.erb](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/master/templates/install_puppet.bat.erb#L42-L44) so that we don't stop mcollective on a Puppet 6 node (before the upgrade).